requirements.txt

```diff
@@ -1,5 +1,5 @@
-django>=1.11,<2.1
-djangorestframework>=3.3,<3.9
+django>=1.11,<3.0
```

Comment:
@nemesisdesign Can we completely drop the support for django<2.0?

Reply:
ok on it

Reply:
oh sorry i have to wait for @nemesisdesign to confirm that we have to drop or not

Reply:
not in this module yet, it may cause problems to someone using this